test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
```

Comment:
>_libManConfig [](start = 33, length = 13)

Doesn't seem like this one needs to be a member variable as all places that access it set it before using it

Reply:
Yes, needs to be reset. Updated in iteration 2.

---
In reply to: [205460826](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205460826) [](ancestors = 205460826)